### PR TITLE
Update docs to refer currently supported Boostrap v5

### PR DESCRIPTION
**Pull request by Alchemik**
JHipster has migrated to Bootstrap v5 some time ago (https://github.com/jhipster/generator-jhipster/pull/16864, https://github.com/jhipster/generator-jhipster/pull/14957), but documentation is still referring to old version. In this PR, for archiving purposes I "detached" the old page from side-menu in the same way as [customizing-bootstrap-3.mdx](https://github.com/jhipster/jhipster.github.io/blob/e7849f8ad1ed7473282637cd1edb43905cc44ed3/docs/customizing-bootstrap-3.mdx#L4), added warning about EOL and created a new page (which at this moment is basically 1:1 copy).
image

### CODE REVIEW

**Comment by mraible** [comment]
@Alchemik Thanks for your contribution! I'd recommend we only have one "Customizing Bootstrap" page and we don't specify the version. Then, we can remove the v4 page. Developers can find the old versions of the page through our [documentation-archive](https://www.jhipster.tech/documentation-archive/).

**Comment by Alchemik** [comment]
@mraible thanks, that makes sense

**Comment by mraible** [comment]
I'd suggest renaming the page as well, to just `customizing-bootstrap.mdx`.

**Comment by Alchemik** [comment]
done

File: docs/development/customizing-bootstrap.mdx [hunk 1]
```diff
@@ -24,6 +25,6 @@ For example to use the border-radius mixin:
 
 Make sure you import only partials and not main Sass files, otherwise you will end up generating duplicate CSS which might cause issues.
 
-To change the default Bootstrap settings like colors, border-radius, etc, add or change the value of the property in the partial file `src/main/webapp/content/scss/_bootstrap-variable.scss`
+To change the default Bootstrap settings like colors, border-radius, etc., add or change the value of the property in the partial file `src/main/webapp/content/scss/_bootstrap-variables.scss`
 
 All values defined in Bootstrap [_variables.scss](https://github.com/twbs/bootstrap/blob/v4-dev/scss/_variables.scss) can be overwritten here.
```
Comment (mraible) [inline docs/development/customizing-bootstrap.mdx:30]: This should link to the v5 version of `_variables.scss`.

**Review by mraible — CHANGES_REQUESTED** [review]

File: docs/development/customizing-bootstrap.mdx (hunk 1 above)
Comment (Alchemik) [reply docs/development/customizing-bootstrap.mdx]: ah, I missed that - fixed

**Comment by mraible** [comment]
@Alchemik Can you please fix the conflict?

**Comment by Alchemik** [comment]
> @Alchemik Can you please fix the conflict?

I think I messed something up, because I cannot resolve conflicts via Github UI and locally the branch is clean:
image
Any ideas? I'm used to Bitbucket and kinda new to Github

**Review by mraible — APPROVED** [review]

**Comment by mraible** [comment]
Thanks for your contribution, @Alchemik!

**Pull request merged**